packages/client/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@bigtest/client",
+  "version": "0.1.0",
+  "description": "Communicate with a BigTest Orchestrator",
+  "main": "dist/cjs/index.js",
+  "browser": "dist/esm/index.js",
+  "typings": "dist/types/index.d.js",
+  "repository": "https://github.com/thefrontside/bigtest.git",
+  "author": "Frontside Engineering <engineering@frontside.com>",
+  "license": "MIT",
+  "files": [
+    "dist/*",
+    "README.md"
+  ],
+  "scripts": {
+    "lint": "eslint '{src,test}/**/*.ts'",
+    "mocha": "mocha -r ts-node/register",
+    "test": "mocha -r ts-node/register test/**/*.test.ts",
+    "prepack": "tsc --outdir dist/esm --sourcemap --module es2015 && tsc --outdir dist/cjs --sourcemap --module commonjs && tsc --outdir dist/types --emitDeclarationOnly"
```

Comment:
should we be using rollup rather than `tsc` direct?

Reply:
Yes, we're using [`tsdx` in `effection`](https://github.com/thefrontside/effection/blob/v0/packages/channel/package.json#L17). I thought we're using it everywhere but it doesn't look like it. I'll make an issue to switch it.

Reply:
Thanks! I just copied this package from `@bigtest/globals` @taras when you create that ticket can you create one to do globals as well.